tasks/need_fixing_linters/need_fixing_linters.py

```diff
@@ -0,0 +1,58 @@
+"""
+Invoke tasks to fix the linter
+"""
+
+from invoke import task
+from invoke.exceptions import Exit
+from ..libs.pipeline_notifications import GITHUB_SLACK_MAP
+from .golangci_lint_parser import filter_lints, display_result, merge_results
+
+FIRST_COMMIT_HASH = "52a313fe7f5e8e16d487bc5dc770038bc234608b"
+# See https://go.dev/doc/install/source#environment for all available combinations of GOOS x GOARCH.
+CI_TESTED_OS_AND_ARCH = {"linux": ["amd64", "arm64"], "windows": ["amd64"], "darwin": ["amd64", "arm64"]}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Does the CI test Darwin arm64 ?